tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -959,6 +960,63 @@ def test_count_miserable_alias_field(self):
         data = response.data["data"]
         assert data[0]["count_miserable_user_300"] == 2
 
+    def test_too_many_transaction_thresholds(self):
+        project_transaction_thresholds = []
+        project_ids = []
+        for i in range(MAX_QUERYABLE_TRANSACTION_THRESHOLDS + 1):
```

Comment:
Nit: Can we mock this constant so we don't have to create 500 projects for this test?

Reply:
good call